server/controller/src/main/java/ai/starwhale/mlops/schedule/k8s/K8sJobTemplate.java

```diff
@@ -69,10 +69,6 @@ public K8sJobTemplate(
         this.pipCacheHostPath = pipCacheHostPath;
     }
 
-    public List<V1Container> getInitContainerTemplates() {
```

Comment:
just delete init containers in the template works too.

Reply:
done